index.html

```diff
@@ -45,6 +45,33 @@ <h2>Connect</h2>
             </ul>
         </section>
         
+        <section>
+            <h2>Leave a Message</h2>
+            <form name="leave_message">
+                <table>
+                    <tr>
+                        <td><label for="name">Name:</label></td>
+                        <td><input type="text" name="name" required="true"></td>
+                    </tr>
+                    <tr>
+                       <td><label for="email">Email:</label></td>
+                        <td><input type="email" name="email" required="true"></td>
+                    </tr>
+                    <tr>
+                        <td><label for="message">Message:</label></td>
+                        <td><textarea name="message" required="true"></textarea></td>
+                    </tr>
+                    <button type="submit">Submit</button>
+                </table>    
+            </form>
+        </section>
+
+        <section id="messages">
+            <h2>Messages</h2>
+            <ul>
```

Comment:
LGTM, did you know that sometimes, if you write u1 for unordered list instead of ul, you would get index.js:1 Warning: The tag <u1> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.